### PR TITLE
Unified commands

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

**Comment by deep-quality-dev** [comment]
@teghnet  Since we deprecate the legacy version of Gofer, I think we can remove [`Gofer`](https://github.com/chronicleprotocol/oracle-suite/pull/183/files#diff-91869ec56d794b63f0381ddf72754d9c3ae322199246dc3766954e17df2d2937R44) in `Config` in `pkg/config/ghostnext/ghost.go`

**Comment by teghnet** [comment]
@deep-quality-dev removed

File: cmd/command.go [hunk 1]
```diff
@@ -13,37 +13,69 @@
...
 				return err
 			}
+			ctx, _ := signal.NotifyContext(context.Background(), os.Interrupt)
 			if err = services.Start(ctx); err != nil {
 				return err
 			}
 			return <-services.Wait()
 		},
 	}
-	c.PersistentFlags().BoolVar(&opts.GoferNoRPC, "gofer.norpc", false, "disable the use of Graph RPC agent")
```
Comment (deep-quality-dev) [inline cmd/command.go:47]: As we remove `--gofer.norpc` command, we need to update `cmd/ghost/README.md`

**Review by jar-o — APPROVED** [review]
much deletion, so wow. I am +1 on tests getting to green

**Pull request merged**